bcipy/signal/model/gaussian_mixture/gaussian_mixture.py

```diff
@@ -104,24 +107,45 @@ def evaluate(self, test_data: np.ndarray, sym_pos: np.ndarray):
 
         return means, covs
 
-    def predict(self, test_data: np.ndarray):
+    def predict(self, test_data: np.ndarray, means: np.ndarray, covs: np.ndarray):
         '''
         Compute log-likelihood of each sample.
         Predict the labels for the test data.
         '''
         # Compute over log-likelihood scores
         # Get the argmax of the scores
 
-        scores = self.model.score_samples(test_data)
+        num_components = len(means)
+
+        N, D = test_data.shape
```

Comment:
No single letter variables if possible, that violates PEP

Reply:
Updated the variable names